File: Lession 1/FindMax.py
def FindMax(A):
    max = A[0]
    for i in range(1, len(A)):
        if A[i] > max:
            max = A[i]
    return max
def main():
    A = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    print(FindMax(A))
if __name__ == "__main__":
    main()

File: Lession 1/FindMax.jl
function FindMax(A::Vector{Int})
    max = A[1]
    for i in 2:length(A)
        if A[i] > max
            max = A[i]
        end
    end
    return max
end
function main()
    A = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
    println(FindMax(A))
end

main()

